student.json

```diff
@@ -0,0 +1 @@
+[{"Name":"George","Age":"41","ID":733}]
```

Comment:
- [ ] Based on the requirements, you should have `JSON` files that store **_books, persons and rentals_** but you have `JSON` files that store for **_students and teachers_**. Kindly store all `Teacher` and `Student` instances in one `JSON` file. 👍 💻

Reply:
Teacher and Student instances are stored in people.json file

Reply:
Hi @theodoku, 

#### This is the JSON file that stores data for Teacher instances:

![Screenshot_1](https://github.com/theodoku/OOP-school-library/assets/77106962/881fb7a1-38be-43b7-9374-2a1cca7e5bbf)

#### This is the JSON file that stores data for Student instances:

![Screenshot_2](https://github.com/theodoku/OOP-school-library/assets/77106962/08051fe4-4258-4101-b167-a972518d51e2)

#### This is the file structure of your repository for this branch, there is no people.json file in it:

![Screenshot_3](https://github.com/theodoku/OOP-school-library/assets/77106962/ad1124b9-3a28-45b6-8da5-a14a478dfb6b)